test/integration/sagemaker/test_mnist.py

```diff
@@ -19,24 +19,36 @@
 import sagemaker
 from sagemaker.pytorch import PyTorchModel
 
-from test.integration import mnist_script, model_cpu_dir
+from test.integration import model_cpu_dir, mnist_cpu_script, mnist_gpu_script, model_eia_dir, mnist_eia_script
 from test.integration.sagemaker.timeout import timeout_and_delete_endpoint
 
 
-@pytest.mark.skip_gpu
-def test_mnist_distributed_cpu(sagemaker_session, ecr_image, instance_type):
+@pytest.mark.cpu_test
+def test_mnist_distributed_cpu(sagemaker_session, ecr_image, instance_type, processor):
     instance_type = instance_type or 'ml.c4.xlarge'
-    _test_mnist_distributed(sagemaker_session, ecr_image, instance_type)
+    _test_mnist_distributed(sagemaker_session, ecr_image, instance_type, processor, mnist_cpu_script)
 
 
-@pytest.mark.skip_cpu
-def test_mnist_distributed_gpu(sagemaker_session, ecr_image, instance_type):
+@pytest.mark.gpu_test
+def test_mnist_distributed_gpu(sagemaker_session, ecr_image, instance_type, processor):
     instance_type = instance_type or 'ml.p2.xlarge'
-    _test_mnist_distributed(sagemaker_session, ecr_image, instance_type)
+    _test_mnist_distributed(sagemaker_session, ecr_image, instance_type, processor, mnist_gpu_script)
 
 
-def _test_mnist_distributed(sagemaker_session, ecr_image, instance_type):
-    model_dir = os.path.join(model_cpu_dir, 'model_mnist.tar.gz')
+@pytest.mark.eia_test
+def test_mnist_eia(sagemaker_session, ecr_image, instance_type, processor, accelerator_type):
+    instance_type = instance_type or 'ml.c4.xlarge'
+    _test_mnist_distributed(sagemaker_session, ecr_image, instance_type, processor, mnist_eia_script, accelerator_type)
```

Comment:
When using keyword function parameters, could you please add the parameter name, like:
`_test_mnist_distributed(sagemaker_session, ecr_image, instance_type, processor, mnist_eia_script, accelerator_type=accelerator_type)`

Helps prevent confusion later on if order of parameters is changed in function definition upon adding any new parameters.

Reply:
Fixed